File: testprojects/master_python_mainone/pipelines/StreamingAllPipeline/code/streamingallpipeline/streaming/main/one/this/graph/Reformat_3.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from streamingallpipeline.streaming.main.one.this.config.ConfigStore import *
from streamingallpipeline.streaming.main.one.this.udfs.UDFs import *

def Reformat_3(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: testprojects/master_scala_mainone/pipelines/SCALA_DEPENDENCY_MANAGEMENT/code/src/main/scala/graph/Reformat_3.scala
package graph

import io.prophecy.libs._
import udfs.UDFs._
import config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_3 {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.select(col("`c   short  --`").as("c   short  --"),
              col("`c   short  --`").as("c_next")
    )

}
